### ISSUE TITLE
Making good things worse (Slint issues)

### ISSUE THREAD

**Issue by Abuseware** [flukejones/asusctl#34]
Hello,

TL;DR: It's not a bug per se, more an user experience on new releases.

Why there was a switch from *egui* to *slint* for rog-control-center? I get that egui is somewhat specific while programming, but it's like whole Rust ecosystem. Slint on the other hand is making build slooow, even with sccache, it's barely readable, things like confirmation of setting change overlays on top of other elements being pain in the ..., it adds only negative experience, and on top of that it's GPL3, which makes me believe it can be somewhat problematic with MPL2 of this project.

So please, please, go back to egui or just take more streamlined approach and pick some GTK/Qt/whatever which will not force me to build entire bible of code just to get small tool up to date. I would rather use gui looking as it came from windows 95 than this, I would pick any web based UI over this.

Everything else is just great, thank you for adding new features!

**Comment by flukejones** [comment]
I changed it because it makes things very easy to prototype and fits with my architecture much better. It has a number of very helpful tools. `egui` was a continuous change-compile-test cycle and took much longer to get things done with
